src/homeblks/volume/tests/vol_gtest.cpp

```diff
@@ -2758,6 +2795,9 @@ int main(int argc, char* argv[]) {
     gcfg.app_mem_size_in_gb = SISL_OPTIONS["app_mem_size_in_gb"].as< uint32_t >();
     gcfg.vol_copy_file_path = SISL_OPTIONS["vol_copy_file_path"].as< std::string >();
     const auto io_size_in_kb = SISL_OPTIONS["io_size"].as< uint32_t >();
+    gcfg.p_zero_buffer = SISL_OPTIONS["p_zero_buffer"].as< uint32_t >();
```

Comment:
as we discussed, would suggest to add a basic zero write/read test case:
1. populate real data from LBA [0, 2MB]
2. write all zeros to [128K, 128K + 32K]
3. read LBA from [128K, 128K + 32K], verify all zeros are returned.

Another basic test:
Similar as above one, only skip the 1st step, e.g. write zeros to LBAs that has never been written before. 

This is to verify that new writes and/or overwrite with all zeros both works.